sdks/python/apache_beam/io/bigquery_test.py

```diff
@@ -291,6 +323,25 @@ def test_read_from_query_sql_format(self):
     self.assertEqual(actual_rows, expected_rows)
     self.assertEqual(schema, reader.schema)
     self.assertFalse(reader.use_legacy_sql)
+    self.assertTrue(reader.flatten_results)
```

Comment:
Please add tests that make sure we retry various requests to BQ and handle timeouts properly.